explorer/ast/declaration.h

```diff
@@ -179,31 +180,40 @@ class SelfDeclaration : public Declaration {
   auto value_category() const -> ValueCategory { return ValueCategory::Let; }
 };
 
+enum class ClassPrefix { StandardClass, BaseClass, AbstractClass };
```

Comment:
```suggestion
enum class ClassExtensibility { None, Base, Abstract };
```

I think this should probably be "None, Base, Abstract" because (with `enum class`) the "Class" suffix feels a little repetitive. But I also think "Prefix" isn't quite capturing the state: [looking at the design wording](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/classes.md#inheritance), maybe "Extensibility" capture this better?

Suggestion is merging these two thoughts, though will obviously require more code edits.

Reply:
I agree with that. By `merging these two thoughts` I understand renaming the enum plus updating its values, is that what you meant?

Reply:
And done.

Reply:
Yes, that's what I meant, thanks. :)